tests/test_strategy_load_hours.py

```diff
@@ -112,9 +112,9 @@ def get_bids(self):
         return deepcopy(self.bids)
 
     def bid(self, price: float, energy: float, buyer: str, original_bid_price=None,
-            buyer_origin=None) -> Bid:
+            buyer_origin=None, replace_existing=False) -> Bid:
```

Comment:
I believe that this testing strategy is not optimal: we are currently creating a mock object that behaves as we want, but this doesn't necessarily mirrors the real implementation of a market. It might become material for a future task.